sphinx/deprecation.py

```diff
@@ -6,10 +6,6 @@
 from typing import Any, Dict, Type
 
 
-class RemovedInSphinx50Warning(DeprecationWarning):
-    pass
-
-
 class RemovedInSphinx60Warning(PendingDeprecationWarning):
```

Comment:
Note: We need to change this to a child of `DeprecationWarning`
```suggestion
class RemovedInSphinx60Warning(DeprecationWarning):
```